### PR TITLE
parser for coala

### PR DESCRIPTION
**Pull request by beaglez**

### CODE REVIEW

File: inlineplz/linters/config/.coafile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[all]
+bears = LineLengthBear
+files = *
```
Comment (guykisel) [inline inlineplz/linters/config/.coafile]: does this mean it'll enforce line length on all files?

**Comment by guykisel** [comment]
this fixes #232

File: inlineplz/linters/config/.coafile (hunk 1 above)
Comment (beaglez) [reply inlineplz/linters/config/.coafile]: Yes, if the project does not specify a dotfile. Basically coala blows up if `bears` and `files` are not specified, so I wanted to put something in there. I've already omitted coala from autorun, perhaps we want to leave it disabled as well?

**Comment by guykisel** [comment]
@beaglez i would actually like if we could turn on a bunch of linters by default in coala, as long as theyre not too annoying, and let people opt-out. the idea of the lint bot is to make static analysis easy with reasonable defaults and little to no configuration :)

**Pull request merged**